### DISCUSSION TITLE
Anybody using Starlink and seeing dead slow speeds with yfinance?

### DISCUSSION THREAD

**Discussion by BiggRanger** [ranaroussi/yfinance#1891] [General]
I've noticed when I'm on Starlink my scraping speeds are around 3 minutes per ticker now, over DSL its a couple of seconds. I'm guessing that Yahoo is seeing the single Starlink gateway and rate limiting everything from it (mine comes up as Chicago , IL).

Anybody else running into the same issue?

**Comment by mberube9** [comment]
Having similar issue.  And it's been on and off for a few days.  Once in a while, my yfinance query takes multiple minutes to complete and I cannot figure out why as I am not really querying enough data to trigger rate limiting.   I didn't think about the Starlink possible issue but that make sense as I am on Starlink as well.

**Comment by maurozac** [comment]
Same issue here. I've started to use Starlink today and immediately noted the slowdown in yfinance queries.

**Comment by barias-lealco** [comment]
I got the same problem using Stalink

**Comment by maurozac** [comment]
I find the problem is the bad DNS server of Starlink. Just change to 1.1.1.1. to fix this.
